sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4864,6 +4960,12 @@ public partial class ScaleInPolicy
         public static bool operator !=(Azure.ResourceManager.Compute.Models.SecurityEncryptionType left, Azure.ResourceManager.Compute.Models.SecurityEncryptionType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class SecurityPostureReference
+    {
+        public SecurityPostureReference() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Compute.VirtualMachineExtensionData> ExcludeExtensions { get { throw null; } }
+        public string Id { get { throw null; } set { } }
```

Comment:
is this in a format of arm resource id? If so, we should change the format to `arm-id`

Reply:
it is "The security posture reference id in the form of". I feel like arm-id sounds a little off. 
Since we updated SecurityPostureReference, to ComputeSecurityPostureReference, can we keep this "id"?

Reply:
i think referenceId sounds better than arm-id

Reply:
well I did not say we would change the name of this property.
The question is: would the value of this property in real traffic always an arm resource id (like an id of a virtual machine `/subscriptions/.../resourceGroups/.../providers/.../etc`)? because from its name, it looks like one.
We do not have to change the name of this property. And if we are not quite sure if this is always an arm id, it is OK that we leave this as a string.

Reply:
yes it looks like it will always be arm-id. what do i need to change then? @ArcturusZhang

Reply:
figured it out. updated to arm-id